actors/builtin/miner/policy_test.go

```diff
@@ -0,0 +1,13 @@
+package miner_test
+
+import (
+	"testing"
+
+	"github.com/filecoin-project/specs-actors/actors/builtin/miner"
+)
+
+func TestWPoStPeriodDeadlines(t *testing.T) {
```

Comment:
This test is a bit redundant as we already check this on `init`, but decided to add it anyways. Let me know if I should remove.

Reply:
We don't need the test, but thanks.